open_seq2seq/encoders/w2l_encoder.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2018 NVIDIA Corporation
```

Comment:
Can you please replace all tabs with spaces (each tab = 2 spaces), so that the code style of this file is consistent with other files?